packages/lodestar/src/validator/nodejs.ts

```diff
@@ -0,0 +1,57 @@
+import defaultConfig, {IValidatorClientOptions} from "./options";
```

Comment:
Probably for a different PR:
I think this should live in the validator package.
Looks like we will probably need to split out our lodestar modules even more if we want to do that though.

Reply:
if we rename lodestar to lodestar-beacon, and add a "convenience" lodestar package that re-exports everything, this code can/should go in the validator package.
Ofc needs the logger and db broken into packages.